### PR TITLE
Typo & duplicated word

### PR DESCRIPTION
**Pull request by kant**
on #7, #89, #159,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ When the reader has completed this Code Pattern, they will understand how to:
 
 ## Included components
 
-* [Watson Visual Recognition](https://www.ibm.com/watson/developercloud/visual-recognition.html): Visual Recognition understands the contents of images - visual concepts tag the image, find human faces, approximate age and gender, and find similar images in a collection.
+* [Watson Visual Recognition](https://www.ibm.com/watson/developercloud/visual-recognition.html): Visual Recognition understands the contents of images - visual concepts: tag the image, find human faces, approximate age and gender, and find similar images in a collection.
```
Comment (scottdangelo) [inline README.md]: I think this needs more than a colon to fix it. Here's something from the website that looks good:
```
Visual Recognition understands the contents of images. Analyze images for scenes, objects, faces, colors, food, text, explicit content, and other subjects that can give you insights into your visual content.
``

File: README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ This code pattern contains several pieces. The app server communicates with the
 
 1. Press the above ``Deploy to IBM Cloud`` button and then click on ``Deploy`` and then jump to step #5.
 
-If you are interested in the deploy to of the application, in Toolchains, click on Delivery Pipeline to watch while the app is deployed.
+If you are interested in the deploy of the application, in Toolchains, click on Delivery Pipeline to watch while the app is deployed.
```
Comment (scottdangelo) [inline README.md]: Better:
```
To monitor the deployment, in Toolchains click on `Delivery Pipeline`  and view the logs while the apps is being deployed.

File: README.md [hunk 3]
```diff
@@ -86,12 +86,12 @@ $ cd watson-vehicle-damage-analyzer
...
   * [**Watson Visual Recognition**](https://console-regional.ng.bluemix.net/catalog/services/visual-recognition)
 
-## 3. Add Visual Recoginition API key to .env file
+## 3. Add Visual Recognition API key to .env file
 
 When you create the Visual Recognition service, you will need the API key
 ![](doc/source/images/apiKey.png)
 
-Move the watson-vehicle-damage-analyzer/server/env.example file to ``/.env`` and populate the API key:
+Move the `watson-vehicle-damage-analyzer/server/env.example file to ``/.env`` and populate the API key:
```
Comment (scottdangelo) [inline README.md]: Missing ending backpack. Should be:
Move the `watson-vehicle-damage-analyzer/server/env.example` file to ``/.env`` and populate the API key:

**Comment by stevemart** [comment]
see https://github.com/IBM/watson-vehicle-damage-analyzer/pull/41/files

**Pull request closed without merging**